test/KwentaNFT.Hardhat.test.ts

```diff
@@ -0,0 +1,298 @@
+
+        it(`should not have gave each tier 1 recipient any tier 0, 2, nor 3 KwentaNFTs`, async () => {
+          for (let account = 0; account < 49; account++) {
+            const balance0 = await KwentaNFT.balanceOf(_to[TIER_1][account], 0)
+            const balance2 = await KwentaNFT.balanceOf(_to[TIER_1][account], 2)
+            const balance3 = await KwentaNFT.balanceOf(_to[TIER_1][account], 3)
+
+            expect(balance0.toNumber()).to.eq(0)
+            expect(balance2.toNumber()).to.eq(0)
+            expect(balance3.toNumber()).to.eq(0)
+          }
+        })
+
+        it(`should not have gave each tier 2 recipient any tier 0, 1, nor 3 KwentaNFTs`, async () => {
+          for (let account = 0; account < 49; account++) {
+            const balance0 = await KwentaNFT.balanceOf(_to[TIER_2][account], 0)
+            const balance1 = await KwentaNFT.balanceOf(_to[TIER_2][account], 1)
+            const balance3 = await KwentaNFT.balanceOf(_to[TIER_2][account], 3)
+
+            expect(balance0.toNumber()).to.eq(0)
+            expect(balance1.toNumber()).to.eq(0)
+            expect(balance3.toNumber()).to.eq(0)
+          }
+        })
+
+        it(`should not have gave each tier 3 recipient any tier 0, 1, nor 2 KwentaNFTs`, async () => {
+          for (let account = 0; account < 5; account++) {
+            const balance0 = await KwentaNFT.balanceOf(_to[TIER_3][account], 0)
+            const balance1 = await KwentaNFT.balanceOf(_to[TIER_3][account], 1)
+            const balance2 = await KwentaNFT.balanceOf(_to[TIER_3][account], 2)
+
+            expect(balance0.toNumber()).to.eq(0)
+            expect(balance1.toNumber()).to.eq(0)
+            expect(balance2.toNumber()).to.eq(0)
+          }
+        })
+      })
+    })
+
+    /**
+    * @dev 5/5 PASS
+    */
+    describe(`uri(uint256 tokenId)`, () => {
+      it(`should get the uri for tier0 tokenIds`, async () => {
+        for (let tokenId = 1; tokenId < 101; tokenId++) {
+          const uri_ = await KwentaNFT.connect(owner).uri(tokenId)
+          expect(uri_).to.eq(uri + '0.json')
+        }
+      })
+
+      it(`should get the uri for tier1 tokenIds`, async () => {
+        for (let tokenId = 101; tokenId < 151; tokenId++) {
+          const uri_ = await KwentaNFT.connect(owner).uri(tokenId)
+          expect(uri_).to.eq(uri + '1.json')
+        }
+      })
+
+      it(`should get the uri for tier2 tokenIds`, async () => {
+        for (let tokenId = 151; tokenId < 201; tokenId++) {
+          const uri_ = await KwentaNFT.connect(owner).uri(tokenId)
+          expect(uri_).to.eq(uri + '2.json')
+        }
+      })
+
+      it(`should get the uri for tier3 tokenIds`, async () => {
+        for (let tokenId = 201; tokenId < 206; tokenId++) {
+          const uri_ = await KwentaNFT.connect(owner).uri(tokenId)
+          expect(uri_).to.eq(uri + '3.json')
+        }
+      })
+    })
+
+    /**
+     * @dev 3/3 PASS
+     */
+    describe(`disableMint()`, () => {
```

Comment:
You would want one more test to ensure that minting is actually disabled. Would look like:
* Contract owner disabled mint
* Contract owner tries to mint
* Contract throws `MintIsDisabled`

But also each tier can only be distributed exactly once, so this whole "disabling mint" is probably unnecessary at this point.

Reply:
You're right, sorry for missing that 😅